File: LeetCode/Easy/Pascal's Triangle/Pascal's Triangle.py
class Solution:
    def generate(self, numRows: int) -> List[List[int]]:
        
        answer = []
        
        for i in range(1,numRows+1):
            tmp = []
            for j in range(i):
                if j == 0 or j == i-1:
                    tmp.append(1)
                else:
                    tmp.append(answer[i-2][j-1]+answer[i-2][j])
            answer.append(tmp)
        
        return answer

File: LeetCode/Easy/Pascal's Triangle/Pascal's Triangle.go
func generate(numRows int) [][]int {
    ans:=[][]int{}
    for i:=0; i<numRows; i++{
        for j:=0; j<i+1; j++{
            if j==0{
                ans = append(ans, []int{1})
            }else{
                if j==i{
                    ans[i] = append(ans[i], 1)
                }else{
                    ans[i] = append(ans[i], ans[i-1][j]+ans[i-1][j-1])
                }
            }
        }
    }
    return ans
}
